nvflare/private/fed/app/fl_app_validator.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import Dict, Tuple
+
+from nvflare.apis.app_validation import AppValidator, AppValidationKey
+
+
+class FLAppValidator(AppValidator):
```

Comment:
Change the class name to AppStructureValidator. See my comment on fl_conf.py

Reply:
Changed the class name to DefaultAppValidator.

Because I think the AppValidator has the same meaning as AppStructureValidator.